catalyst-gateway/bin/src/event_db/legacy/queries/event/config.rs

```diff
@@ -0,0 +1,81 @@
+//! Config Queries
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use tracing::error;
+
+use crate::event_db::{Error, EventDB};
+
+#[async_trait]
+#[allow(clippy::module_name_repetitions)]
+/// Config Queries Trait
+pub(crate) trait ConfigQueries: Sync + Send + 'static {
+    async fn get_config(&self) -> Result<(Vec<NetworkMeta>, FollowerMeta), Error>;
+}
+use crate::event_db::Error::JsonParseIssue;
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, PartialOrd, Clone)]
+/// Network config metadata
+pub(crate) struct NetworkMeta {
+    /// Mainnet, preview, preprod
```

Comment:
testnet too

Reply:
Same context in relation to parsing raw JSON from the DB and doing conversions and stuff. It's easier to leave it.